ui/raidboss/data/triggers/o8n.js

```diff
@@ -6,110 +6,401 @@
     {
       id: 'O8N Hyper Drive',
       regex: /14:292E:Kefka starts using Hyper Drive on (\y{Name})/,
+      regexDe: /14:292E:Kefka starts using Hyperantrieb on (\y{Name})/,
+      regexFr: /14:292E:Kefka starts using Colonne De Feu on (\y{Name})/,
       alertText: function(data, matches) {
         if (matches[1] == data.me)
-          return 'Tank Buster on YOU';
+          return {
+          en: 'Hyperdrive on YOU',
+          fr: 'Colonne de feu sur VOUS',
+          de: 'Hyperantrieb auf DIRS',
+        };
         if (data.role == 'healer')
-          return 'Buster on ' + data.ShortName(matches[1]);
+          return {
+            en: 'Hyperdrive on ' + data.ShortName(matches[1]),
+            fr: 'Colonne de feu on ' + data.ShortName(matches[1]),
+            de: 'Hyperantrieb auf ' + data.ShortName(matches[1]),
+          };
       },
       tts: function(data, matches) {
         if (matches[1] == data.me)
-          return 'buster';
+          return {
+            en: 'buster',
+            fr: 'Colonne de feu',
+            de: 'hyperantrieb',
+          };
       },
     },
     {
       id: 'O8N Shockwave',
       regex: /14:2927:Graven Image starts using Shockwave/,
+      regexDe: /14:2927:Heilige Statue starts using Schockwelle/,
+      regexFr: /14:2927:Statue Divine starts using Onde De Choc/,
       delaySeconds: 5,
-      alertText: 'Look for Knockback',
-      tts: 'knockback',
+      alertText: {
+        en: 'Look for Knockback',
+        fr: 'Se préparer à la projection',
+        de: 'Auf Rückstoß achten',
+      },,
```

Comment:
Can you remove one of these commas?

The thing I always do when submitting triggers is to reload raidboss and make sure that it doesn't have any errors.

Reply:
Oh, I didn't see that. Gonna fix it :)